src/branding/default/branding.desc

```diff
@@ -219,3 +219,18 @@ slideshow:               "show.qml"
 slideshowAPI: 2
 
 
+# These options are to customize online uploading of logs to pastebins:
+#  - style  :  Defines the kind of pastebin service to be used. ie it can
```

Comment:
style key is not hooked up to anywhere in backend.

Reply:
that the 'enable' key is used but the 'style' key is not, indicates a smell - i would eliminate the 'enable' key - that semantics could be deduced by the value of 'style' (eg: 'none' or empty-string) - the only utility i can imagine for an 'enable' key, would be if there existed a mechanism to enable/disable the feature at runtime

i would also suggest the renaming those logUpload* keys to: uploadServer* and the 'style' key name to: 'type' instead, to be more meaningful